tests/cc/usdt_test_lib.c

```diff
@@ -0,0 +1,10 @@
+#ifdef HAVE_SDT_HEADER
+#include <sys/sdt.h>
+
```

Comment:
Some of our test machines may not have sys/sdt.h. Maybe you can use the folly headers? Currently, it is in bcc/tests/python/include. But I think you can add a symbolic link at bcc/tests/cc to use it.

Reply:
addressed, and removed `sys/sdt.h` requirement from another test